controllers/news.controller.js

```diff
@@ -106,3 +105,17 @@ exports.postNewComment = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getAllUsers = (req, res) => {
+  const table = "users";
+  receiveAll(table).then((data) => {
+    data.forEach(item =>{
+      if(!item.username || !item.name || !item.avatar_url){
+        return Promise.reject({code: 404, msg: 'not found'})
+      }
+    })
+    res.status(200).send({ [table]: data });
+  }).catch(err=>{
+    next(err)
+  });
+};
```

Comment:
I really like the changes you have made to make your receiveAll function totally reusable by just passing in the table name. Really nice solution!